### PR TITLE
RS/YJ/Rule 22-17

### PR DESCRIPTION
**Pull request by yunjoonjung-PNNL**
This didn't fully pass with the TCD, but can be reviewed now. I'll work with @jugonzal07 to make this rule pass. Thanks.

### CODE REVIEW

File: rct229/rulesets/ashrae9012019/section22/section22rule17.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+                    f"*Note: The fan shaft power for {heat_rejection_b['id']} was not given. For this evaluation, the fan shaft power was calculated using a rule of thumb "
+                    f"where fan_shaft_power = fan_motor_nameplate_power * {FAN_SHAFT_POWER_FACTOR} * fan_motor_efficiency."
+                )
+            )
+            heat_rejection_efficiency_b = calc_vals["heat_rejection_efficiency_b"]
+            heat_rejection_efficiency_in_gpm_per_hp_b = round(
+                heat_rejection_efficiency_b.to(ureg("gpm/hp")).magnitude, 1
+            )
+
+            if std_equal(HEAT_REJ_EFF_LIMIT, heat_rejection_efficiency_b):
```
Comment (weilixu) [inline rct229/rulesets/ashrae9012019/section22/section22rule17.py:105]: I think this case, we should have a strict equal

File: rct229/rulesets/ashrae9012019/section22/section22rule17.py [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+                UNDETERMINED_MSG = (
```
Comment (weilixu) [inline rct229/rulesets/ashrae9012019/section22/section22rule17.py:106]: convention is using a lower case variable name: `undetermined_msg`

**Review by weilixu — APPROVED** [review]
Approved with two comments to address

File: rct229/rulesets/ashrae9012019/section22/section22rule17.py (hunk 2 above)
Comment (yunjoonjung-PNNL) [reply rct229/rulesets/ashrae9012019/section22/section22rule17.py:106]: Addressed.

File: rct229/rulesets/ashrae9012019/section22/section22rule17.py (hunk 1 above)
Comment (yunjoonjung-PNNL) [reply rct229/rulesets/ashrae9012019/section22/section22rule17.py:105]: Addressed.

**Pull request merged**